examples/contour/01-contour-config.yaml

```diff
@@ -11,6 +11,11 @@ data:
     #   determine which XDS Server implementation to utilize in Contour.
     #   xds-server-type: contour
     #
+    # specify the service-apis Gateway Contour should configure
```

Comment:
I think it makes sense to be at top level since this will be guarded behind the service APIs flag anyways right?

Reply:
This is the design I have in the design document - I did it this way so that this being non-nil can replace the boolean flag. I don't think there's a need for the flag when setting a value performs the same function.

Reply:
Nit:
```suggestion
    # Specify the service-apis Gateway Contour should watch.
```

Reply:
Yeah this was just setting up the groundwork to allow the Gateway to be configured here. Once this is in we can start to utilize it in the code without breaking anything else.